### DISCUSSION TITLE
Regarding the modeling details in the paper

### DISCUSSION THREAD

**Discussion by chrissymkcn** [BayraktarLab/cell2location#324] [General]
Thank you again for the wonderful tool. I just would like to ask about some parts of the modeling details. 

My question is What does the total special abundance z_s,r mean exactly? My understanding is that it is a latent cell group specific (I.e. varying across r in {1…R}) measure, the formula goes like zs,r ∼ Gamma(Bs/R, 1/(Ns/Bs)). However, Bs is the parameter I truly did not understand. Bs is “the latent number of groups r expected in each location s”, which I understand as how many groups chosen among the R groups. Then how is this z_s,r different by r? Or is Bs the number of cells that belong to group r?

Thank you for your clarification.

**Comment by vitkl** [comment]
> Bs is “the latent number of groups r expected in each location s”, which I understand as how many groups chosen among the R groups. 

Correct.

Bs controls the sparsity of Gamma distributions that produces z_{s,r} by defining the shape of the Gamma distribution. Eg if for a given location B_s=5 and R=50, shape=5/50 resulting in a sparse prior distribution over z_{s,r}.

Now, N_s is the expected total cell abundance per location. `Ns/Bs` gives the expected number of cells per location per tissue zone (aka group) and corresponds to scale of the Gamma distribution. You can loosely interpret this as `shape=5/50` leading to only 5 out of 50 non-zero values with the expected value of `Ns/Bs` for those non-zero values.

The above gives a loosely informative prior that keeps the total cell abundance per location not too far from `Ns`.
